pkg/registries/ecr/transport.go

```diff
@@ -46,11 +47,12 @@ func (t *awsTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 	)
 }
 
-func (t *awsTransport) isExpiredNoLock() bool {
-	return t.expiresAt == nil || t.expiresAt.After(time.Now())
+func (t *awsTransport) isValidNoLock() bool {
+	return t.expiresAt != nil && t.expiresAt.After(time.Now())
 }
 
 func (t *awsTransport) refreshNoLock() error {
+	log.Infof("Refreshing ECR token for image integration %q", t.name)
```

Comment:
how often do we expect to see this log (same with google)? is this very often? should we convert to debug log?

Reply:
once every hour per registry for google, once every 12 hours per registry for ecr. I guess `debug` is maybe the safer choice if someone has a lot of registries, it could indeed cause some log spam.